identity/users_test.go

```diff
@@ -39,6 +42,111 @@ func MockIdentityClient() *identity.IdentityClient {
 	}
 }
 
+func TestAccUser_ChangeUserPassword(t *testing.T) {
+	userLogin := testutils.RandPrefixString("TestAccUser", 32)
+	userEmail := fmt.Sprintf("%s@example.com", userLogin)
+	userPassword := testutils.RandString(32)
+	userNewPassword := testutils.RandString(32)
+
+	// Holds newly created user.
+	var newUser *identity.User
```

Comment:
The purpose of the state bag construct is to remove global state from the acceptance testing process. I'll discuss more in a bit.

Reply:
True, and I really like the idea.

I only added this in a bid to pass user ID to an invocation of `StepAPICall` where I call `ChangeUserPassword`. I've noticed that it does not have support for passing the state bag (like the others), and using variable from more top-level scope in the closure was an option to retrieve the ID of newly created user.